### PR TITLE
=str #16652 Add java docs for 'Working with streaming IO'

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #16652

File: akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala [hunk 1]
```diff
@@ -135,7 +129,6 @@ class StreamTcpDocSpec extends AkkaSpec {
       .transform(() => RecipeParseLines.parseLines("\n", maximumLineBytes = 256))
       .map(text => println("Server: " + text))
       .map(_ => readLine("> "))
-      //#repl-client
```
Comment (ktoso) [inline akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala]: +1, thanks

File: akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/StreamTcpDocTest.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (C) 2014 Typesafe Inc. <http://www.typesafe.com>
```
Comment (ktoso) [inline akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/StreamTcpDocTest.java]: minor: bump copyright template to 2015 :)

**Comment by ktoso** [comment]
LGTM

**Comment by drewhk** [comment]
LGTM

**Pull request merged**